apps/wallet-dashboard/app/globals.css

```diff
@@ -19,6 +19,7 @@ body {
 
 @layer components {
     .home-page-grid-container {
+        height: calc(100vh - 130px);
```

Comment:
why 130px? Shouldn't it be 72px?
![image](https://github.com/user-attachments/assets/62057b14-1b35-43db-97e2-a01243d444d6)